extension/llm/export/quantizer_lib.py

```diff
@@ -141,6 +141,40 @@ def check_embedding_byte_registered():
     return quantizers
 
 
+def get_coreml_quantizer(args):
+    try:
+        from coremltools.optimize.torch.quantization.quantization_config import (
+            LinearQuantizerConfig,
+            QuantizationScheme,
+        )
+        from executorch.backends.apple.coreml.quantizer import CoreMLQuantizer
+    except ImportError:
+        raise ImportError(
+            "Please install the CoreML backend follwing https://pytorch.org/executorch/main/build-run-coreml.html"
+        )
+
+    quantization_config = LinearQuantizerConfig.from_dict(
+        {
+            "global_config": {
+                "activation_dtype": "float32",
+            }
+        }
+    )
+    quantization_config = LinearQuantizerConfig.from_dict(
+        {
+            "global_config": {
+                "quantization_scheme": QuantizationScheme.symmetric,
+                "milestones": [0, 0, 10, 10],
+                "activation_dtype": torch.float32,
+                "weight_dtype": torch.qint8,
```

Comment:
I remember we can use regular int8 now, i.e. simple `torch.int8` should work 🙏 Please give it a try to see if we can get rid of q dtype